web/modules/custom/portland/modules/portland_leaflet_maps/portland_leaflet_maps.module

```diff
@@ -75,46 +61,37 @@ function portland_leaflet_maps_leaflet_map_info_alter(&$map_info) {
  * Assemble default map info.
  */
 function _portland_leaflet_maps_assemble_default_map_info(&$map_info, $default_settings = array()) {
-  // Use headless protocol rather than logic based on global $is_https;
-  $prot = '//';
-
-  // As used by most.
-  $zxy = '{z}/{x}/{y}.png';
-  // As used by Esri.
-  $zyx = '{z}/{y}/{x}.png';
-  // Used by PortlandMaps
+  // Coordinate format used by PortlandMaps
   $zyx_only = '{z}/{y}/{x}';
 
-  // portlandmaps https://portlandmaps.com
-  $attr_portlandmaps = 'Tiles by <a href="https://portlandmaps.com">portlandmaps.com</a>. ';
+  $attr_portlandmaps = '<a href="https://www.portlandmaps.com">PortlandMaps</a>';
 
   $map_info['portlandmaps_color'] = array(
     'label' => 'PortlandMaps.com Basemap_Color_Complete (zoom 0..22)',
     'description' => t('portlandmaps.com Basemap_Color_Complete Maps'),
-    'settings' =>  $default_settings + array('maxZoom' => 22),
-    'layers' => array(
-      'layer' => array(
-        'urlTemplate' => $prot . "www.portlandmaps.com/arcgis/rest/services/Public/Basemap_Color_Complete/MapServer/tile/$zyx_only",
-      ),
-    ),
+    'settings' =>  $default_settings + ['maxZoom' => 22],
+    'layers' => [
+      'layer' => [
+        'urlTemplate' => "https://www.portlandmaps.com/arcgis/rest/services/Public/Basemap_Color_Complete/MapServer/tile/$zyx_only",
+        'options' => [
+          'attribution' => $attr_portlandmaps,
+        ],
+      ],
+    ],
   );
 
   $map_info['portlandmaps_park'] = array(
     'label' => 'PortlandMaps.com Park_Amenities (zoom 0..22)',
     'description' => t('portlandmaps.com Park_Amenities Maps'),
-    'settings' =>  $default_settings + array('maxZoom' => 22),
-    // The order is important. From bottom to top: color basemap, building outlines, park amenities (only available at a few zoom levels)
-    'layers' => array(
-      'layer #1' => array(
-        'urlTemplate' => $prot . "www.portlandmaps.com/arcgis/rest/services/Public/Basemap_Color_Complete/MapServer/tile/$zyx_only",
-      ),
-      'layer #2' => array(
-        'urlTemplate' => $prot . "www.portlandmaps.com/arcgis/rest/services/Public/Basemap_Color_Buildings/MapServer/tile/$zyx_only",
-      ),
-      'layer #3' => array(
-        'urlTemplate' => $prot . "www.portlandmaps.com/arcgis/rest/services/Public/Park_Amenities/MapServer/tile/$zyx_only",
```

Comment:
Looks like the Park_Amenities tile layer has been removed. I'm not able to find a good replacement.

Reply:
Yeah I removed the layer because it appears the CGIS service is 404ing. Coincidentally we recently just got a Zendesk request #400293 asking about more layers on the park finder, so likely we'll be adding some more stuff back in soon.